Decisions/scs-xxxx-v1-ssd-flavors.md

```diff
@@ -0,0 +1,250 @@
+title: SSD Flavors
+type: Decision Record
+status: Draft
+track: IaaS
+version: v0.1
+enhances: scs-xxxx-v1-flavor-naming.md
```

Comment:
There are dashes missing in this line and in the first line.

Adding them enables the Markdown renderer to understand them. Like in SCS-0001: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Standards/scs-0001-v1-sovereign-cloud-standards.md

Reply:
Good catch, thanks. Hopefully it's correct now.